trackpaypal.php

```diff
@@ -216,4 +216,60 @@ function trackpaypal_civicrm_postIPNProcess(&$IPNData) {
   // and transaction details to send to Google Analytics
   // via its REST interface
   watchdog('paypal', 'IPN payload: %payload', array('%payload' => print_r($IPNData, TRUE)), WATCHDOG_DEBUG);
+
+  // Retrieve extension settings
+  $result = civicrm_api3('setting', 'get', ['return' => ['trackpaypal_event_type'],]);
+  $event_type = $result['values']['1']['trackpaypal_event_type'];
```

Comment:
```suggestion
  - $result = civicrm_api3('setting', 'get', ['return' => ['trackpaypal_event_type'],]);
  - $event_type = $result['values']['1']['trackpaypal_event_type'];
  + $event_type = Civi::settings->get('trackpaypal_event_type');
```
This is as per https://docs.civicrm.org/dev/en/latest/framework/setting/#setting-storage-and-retrieval